nl/xfrm_state_linux_test.go

```diff
@@ -205,3 +205,12 @@ func TestXfrmEncapTmplDeserializeSerialize(t *testing.T) {
 	msg := DeserializeXfrmEncapTmpl(orig)
 	testDeserializeSerialize(t, orig, safemsg, msg)
 }
+
+func TestSerializeDeserializeXfrmMark(t *testing.T) {
```

Comment:
No valid reason: I did not pay attention to the existing test code. Thanks to your comment, I just realized there is a template to automate the serialization/deserialization check of the structure filled with random data. 

I will push now the proper test code for the new structure. Thanks.